### ISSUE TITLE
Update abbreviation:eng_x_preferred properties

### ISSUE THREAD

**Issue by stepps00** [whosonfirst-data/whosonfirst-data#905]
The recent GeoNames work introduced a new property, `abbreviation:eng_x_preferred` to various WOF records. Let's shorten this to `abrv:eng_x_preferred`. 

Also, make sure the `whosonfirst-properties` repo is updated with a new name and description for this property.

**Comment by nvkelso** [comment]
Is this documentation correct?

https://github.com/whosonfirst/whosonfirst-properties/blob/dd3ec38951423f156f5775827cfd7898064c9da6/properties/wof.md#abbreviation

**Comment by stepps00** [comment]
Yeah, the `wof:abbreviation` is correct, but a new `abrv.md` will be needed to track `abrv:eng_x_preferred` (and any other variations).

That work will fall under https://github.com/whosonfirst/whosonfirst-properties/issues/33, which I'll use to add .json files to the `whosonfirst-properties` repo.
